common/src/main/scala/com/gu/media/youtube/YouTubePartnerApi.scala

```diff
@@ -171,8 +173,28 @@ trait YouTubePartnerApi { this: YouTubeAccess with Logging =>
     }
   }
 
+  private def updateTheVideoAdvertisingOptions(videoId: String, atomId: String): Either[VideoUpdateError, String] = {
+    val formats: util.List[String] = List("standard_instream","trueview_instream","display").asJava
```

Comment:
Worth adding a comment to the YouTube docs for future reference if we decide to add/remove formats?

Reply:
Yeah, good shout 👍